webxrlayers-1.bs

```diff
@@ -732,13 +729,16 @@ The {{XREquirectLayerInit}} dictionary represents a set of configurable values t
 dictionary XREquirectLayerInit : XRLayerInit {
   XRRigidTransform? transform;
   float radius = 10.0;
-  float scaleX = 1.0;
-  float scaleY = 1.0;
-  float biasX = 0;
-  float biasY = 0;
+  float centralHorizontalAngle = 6.28318;
+  float upperVerticalAngle = 3.14159;
```

Comment:
It can't be PI. It changes from -PI/2 to PI/2. I'd say the default value should be PI/2.
```suggestion
  float upperVerticalAngle = 1.570795;
```